src/Nordea/Components/Tooltip.elm

```diff
@@ -18,6 +18,8 @@ type Placement
 type alias Config msg =
     { placement : Placement
     , content : List (Html msg)
+    , arrowColor : Maybe Css.Color
+    , overrideShow : Maybe Bool
```

Comment:
Looks like `Nothing == False`
```suggestion
    , overrideShow :Bool
```

Reply:
This will make more sense soon. When you override with False here then you don't want the auto hover show functionality